src/box/lua/config/instance_config.lua

```diff
@@ -1118,6 +1118,67 @@ return schema.new('instance_config', schema.record({
             end
         end,
     }),
+    security = schema.record({
+        auth_type = schema.enum({
+            'chap-sha1',
+            'pap-sha256',
+        }, {
+            box_cfg = 'auth_type',
+            default = 'chap-sha1',
```

Comment:
AFAIU, you said that the defaults are different on CE and EE and we agreed to set `default = box.NULL` in the schema with a comment about the meaning of such a value.

Reply:
I think, there was a misunderstanding, the default in both CE and EE [is `chap-sha1`](https://www.tarantool.io/en/enterprise_doc/security/#authentication-protocol), my problem was that I didn't know how to implement validation and you helped me with that.